src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+$calendar-day-width: 100px;
+$calendar-day-gap: 1px;
+$the-last-day: 31;
+$feb-day-count: 28;
+$week:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-day-gap;
+  width: 7 * $calendar-day-width + 16px;
+
+  &__day {
+    box-sizing: border-box;
+    height: $calendar-day-width;
+    width: $calendar-day-width;
+    border: 1px solid black;
+    background-color: #eee;
+    transition: transform 0.3s;
```

Comment:
in description for task The hovered cell has to become pink says:

    (Optional) Move it up by 20px (use transform)
    (Optional) both of these properties are to be animated with the duration of half a second.Both properties and 0.5w

Reply:
it becomes pink

Reply:
> it becomes pink

Yes, it does but it becomes pink immediately.
8. [STYLES] - Make sure to list all styles that you apply transition to.

You need to add color to transition